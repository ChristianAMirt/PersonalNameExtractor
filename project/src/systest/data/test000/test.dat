<NER> Emma Smith, Alexander Johnson? Olivia Brown, William Davis, Sophia Martinez!
James Anderson, Ava Garcia, Benjamin
Wilson, Mia Taylor. Ethan P. Lopez, Samantha-Jane O'Connor. </NER>