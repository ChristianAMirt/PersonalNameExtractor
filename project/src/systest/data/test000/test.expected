<NER> <PER>Emma Smith</PER>, <PER>Alexander Johnson</PER>? <PER>Olivia Brown</PER> <PER>William Davis</PER>, <PER>Sophia Martinez</PER>!
<PER>James Anderson</PER>, <PER>Ava Garcia</PER>, <PER>Benjamin
Wilson</PER>, <PER>Mia Taylor</PER>. <PER>Ethan P. Lopez</PER>, <PER>Samantha-Jane O'Connor</PER>. </NER>